helpers/Matrix.h

```diff
@@ -0,0 +1,46 @@
+#ifndef _HELPERS_MATRIX_H_
+#define _HELPERS_MATRIX_H_
+
+#ifndef _Z64HDR_HELPER_PREFIX_
```

Comment:
```suggestion
#ifndef _Z64HDR_HELPER_QUALIFIERS_
```
I don't think qualifier is the exact term but I can't find one better, still better than prefix though (imo, until I read the readme I thought "prefix" meant prefixing the function name somehow)

Reply:
It is to PREFIX helper functions with static, inline or both if user wants to. I added it as #ifndef so that by default it is #defined with nothing, but if user wants to define like static inline, they can do it easily with this. So like #define _Z64HDR_HELPER_PREFIX_ static inline

Reply:
yeah, and another word is to "qualify" functions with such keywords, more specific than prefix imo